Lib/test/test_asyncio/test_windows_events.py

```diff
@@ -56,6 +56,7 @@ def test_pipe(self):
         res = self.loop.run_until_complete(self._test_pipe())
         self.assertEqual(res, 'done')
 
+    @asyncio.coroutine
```

Comment:
Shouldn't this be converted to 'async def' too?

Reply:
I was trying to minimize tests change for sake of easy review.
One line for adding missing `@asyncio.coroutine` or several `yield from` -> `await` replacements.

Reply:
I think it's OK to upgrade tests too.